tox.ini

```
@@ -36,6 +36,14 @@ skip_install = false
 deps =
     matplotlib==3.8.0
 
+[plot_full_min]
+description = environment with minimum matplotlib, plotly and kaleido versions
+skip_install = false
+deps =
+    matplotlib==3.8.0
```

```suggestion
    {[plotmin]deps}
```

this should work, no?